casper/src/main/scala/coop/rchain/casper/SafetyOracle.scala

```diff
@@ -281,23 +256,30 @@ sealed abstract class SafetyOracleInstances {
                 false.pure[F]
               )
           }
+
+        def findMaximumClique(
```

Comment:
This should move to the `Clique`object and be separately tested. 
It no longer returns the max clique, but rather it's weight